client/extensions/woocommerce/components/form-dimensions-input/index.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
+
+function FormDimensionsInput( {
+	className,
+	dimensions,
+	unit,
+	onChange,
+	translate,
+} ) {
+	const classes = classNames( 'form-dimensions-input', className );
+
+	return (
+		<div className={ classes }>
+			<FormTextInput
+				name="length"
+				placeholder={ translate( 'L' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'L', { context: 'momentjs format string (sameElse)'} )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).